frontend/src/app/App.scss

```diff
@@ -78,3 +78,11 @@ body,
     width: 100%;
   }
 }
+
+// Override for about box height, reduce to fit the content instead of a fixed height
+.pf-v5-c-about-modal-box {
+  height: fit-content;
+  &__content {
+    grid-column-end: -1;
+  }
+}
```

Comment:
Hmmm -- is this necessary? This is to prevent a scrollbar? This seems like something the About Modal from PF should managed, no?

Reply:
This is to allow the text to span more of the dialog. Without it the text and the info fields are limited in the space it can use. Here is a snapshot without this setting:

![image](https://github.com/opendatahub-io/odh-dashboard/assets/11633780/776a424f-6245-499d-ae92-1a8396b0acd8)

The PF team is assessing and will add a modifier to provide this update.